python/ray/rllib/agents/marwil/marwil.py

```diff
@@ -0,0 +1,150 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import time
+
+from ray.rllib import optimizers
+from ray.rllib.agents.agent import Agent, with_common_config
+from ray.rllib.agents.marwil.marwil_policy_graph import MARWILPolicyGraph
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.schedules import ConstantSchedule, LinearSchedule
+
+OPTIMIZER_SHARED_CONFIGS = [
+    "learning_starts", "buffer_size", "prioritized_replay",
+    "prioritized_replay_alpha", "prioritized_replay_beta",
+    "schedule_max_timesteps", "beta_annealing_fraction",
+    "final_prioritized_replay_beta", "prioritized_replay_eps",
+    "train_batch_size", "sample_batch_size"
+]
+
+# yapf: disable
+# __sphinx_doc_begin__
+DEFAULT_CONFIG = with_common_config({
+    # If true, use the Generalized Advantage Estimator (GAE)
+    # with a value function, see https://arxiv.org/pdf/1506.02438.pdf.
+    "use_gae": False,
+    # GAE(lambda) parameter
+    "lambda": 1.0,
+
+    # === Model ===
+    "actor_hiddens": [64],
```

Comment:
I think we should avoid having these types of options, are the model options sufficient?